src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/MemberAndDeclarationInfo.cs

```diff
@@ -246,7 +246,28 @@ private sealed class MemberNameComparer : IComparer<MemberAndDeclarationInfo>
             public int Compare(MemberAndDeclarationInfo x, MemberAndDeclarationInfo y)
             {
                 var comp = string.Compare(x.Name, y.Name, StringComparison.Ordinal);
-                return comp != 0 ? comp : (y._inheritanceLevel - x._inheritanceLevel);
+                if (comp == 0)
+                {
+                    return y._inheritanceLevel - x._inheritanceLevel;
+                }
+
+                // Field members come first
```

Comment:
>Field members come first [](start = 19, length = 24)

This orders fields first rather than `_` members first.

Reply:
Sorry about that, I didn't revert far enough. Last commit should be good